ooiui/static/js/views/common/MapView.js

```diff
@@ -324,7 +325,7 @@ var MapView = Backbone.View.extend({
       }
     });
 
-    markerCluster.on('clustermouseout', function (e) {
```

Comment:
@dlangston where did the marker cluster on move get put? @DanielJMaher didnt we still want that?